swagger.yaml

```diff
@@ -708,6 +708,40 @@ paths:
         "default":
           description: Invalid request was sent to the server.
 
+  "/users/{userId}/severa-opt-in":
+    delete:
+      operationId: removeSeveraOptIn
+      summary: Remove Severa opt-in for user
+      description: Remove the user's Severa opt-in status.
+      tags:
+        - Users
+      parameters:
+        - name: userId
```

Comment:
In severa we look up the user by email, I assume the keycloak Id is enough to allow us to look up the user email in the lambda? This would be preferred to including the email here if possible. And email look up form keycloak id is possible, then removing the email from the update attribute endpoint would also be an improvement to be done in the future task I mentioned in the previous comment